comp/otelcol/ddflareextension/impl/testdata/simple-dd/config-enhanced-result.yaml

```diff
@@ -46,8 +46,8 @@ exporters:
     sending_queue:
       batch:
         flush_timeout: 10s
-        max_size: 0
-        min_size: 8192
+        max_size: 100
+        min_size: 10
```

Comment:
Since you are setting the defaults programmatically (rather than via config files), I don't think the default values are reflected in the yaml files. You can try just removing these, that may fix the tests